src/main/scala/com/cloudera/sparkts/TimeSeriesStatisticalTests.scala

```diff
@@ -312,5 +314,107 @@ object TimeSeriesStatisticalTests {
     val bpstat = residuals.length * auxOLS.calculateRSquared()
     val df = factors.cols // auxOLS uses intercept term, so (# of regressors - 1) = # factors cols
     (bpstat, 1 - new ChiSquaredDistribution(df).cumulativeProbability(bpstat))
-  }   
+  }
+
+  /**
+   * Critical values associated with KPSS test when testing with a simple constant
+   * Source:
+   * Kwiatkowski, D., Phillips, P., Schmidt, P., & Shin, Y.
+   * Testing the null hypothesis of stationarity against the alternative of a unit root.
+   * Journal of Econometrics, 159-178.
+   */
+  private val kpssConstantCriticalValues = ListMap(0.10 -> 0.347, 0.05 -> 0.463, 0.025 -> 0.574,
+    0.01 -> 0.739
```

Comment:
I think it'd look a little nicer to have all the critical values on the same line